projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,1554 @@
+	"item.quark.white_shard": "白色玻璃碎片",
+	"item.quark.yellow_shard": "黄色玻璃碎片",
+	"item.quark.clear_shard": "玻璃碎片",
+	"item.quark.dirty_shard": "脏玻璃碎片",
+	"item.quark.glass_item_frame": "玻璃物品展示框",
+	"item.quark.glowing_glass_item_frame": "荧光玻璃物品展示框",
+	"item.quark.black_rune": "黑色符文",
+	"item.quark.blue_rune": "蓝色符文",
+	"item.quark.brown_rune": "棕色符文",
+	"item.quark.cyan_rune": "青色符文",
+	"item.quark.gray_rune": "灰色符文",
+	"item.quark.green_rune": "绿色符文",
+	"item.quark.light_blue_rune": "淡蓝色符文",
+	"item.quark.light_gray_rune": "淡灰色符文",
+	"item.quark.lime_rune": "黄绿色符文",
+	"item.quark.magenta_rune": "品红色符文",
+	"item.quark.orange_rune": "橙色符文",
+	"item.quark.pink_rune": "粉红色符文",
+	"item.quark.purple_rune": "紫色符文",
+	"item.quark.red_rune": "红色符文",
+	"item.quark.white_rune": "白色符文",
+	"item.quark.yellow_rune": "黄色符文",
+	"item.quark.rainbow_rune": "彩虹符文",
+	"item.quark.blank_rune": "空白符文",
+	"item.quark.pickarang": "皮卡镖",
+	"item.quark.diamond_heart": "钻石之心",
+	"item.quark.ancient_tome": "古卷",
+	"item.quark.crab_bucket": "螃蟹桶",
+	"item.quark.crab_leg": "生螃蟹腿",
+	"item.quark.cooked_crab_leg": "熟螃蟹腿",
+	"item.quark.crab_shell": "螃蟹壳",
+	"item.quark.dragon_scale": "龙鳞",
+	"item.quark.moss_paste": "苔藓泥",
+	"item.quark.backpack": "背包",
+	"item.quark.ravager_hide": "劫掠兽皮",
+	"item.quark.soul_compass": "灵魂指南针",
+	"item.quark.bottled_cloud": "瓶中云",
+	"item.quark.flamerang": "界炎镖",
+	"item.quark.echorang": "回响镖",
+	"item.quark.seed_pouch": "种子袋",
+	"item.quark.forgotten_hat": "遗忘者之帽",
+	"item.quark.soul_bead": "灵魂珠",
+	"item.quark.abacus": "算盘",
+	"item.quark.azalea_boat": "杜鹃木船",
+	"item.quark.blossom_boat": "花木船",
+	"item.quark.azalea_chest_boat": "杜鹃木运输船",
+	"item.quark.blossom_chest_boat": "花木运输船",
+	"item.quark.egg_parrot_red_blue": "红色鹦鹉蛋",
+	"item.quark.egg_parrot_blue": "蓝色鹦鹉蛋",
+	"item.quark.egg_parrot_green": "绿色鹦鹉蛋",
+	"item.quark.egg_parrot_yellow_blue": "青色鹦鹉蛋",
+	"item.quark.egg_parrot_grey": "灰色鹦鹉蛋",
+	"item.quark.music_disc_endermosh": "音乐唱片",
+	"item.quark.music_disc_endermosh.desc": "Kain Vinosec - Endermosh",
+	"item.quark.music_disc_drips": "环境唱片",
+	"item.quark.music_disc_drips.desc": "Water Droplets",
+	"item.quark.music_disc_ocean": "环境唱片",
+	"item.quark.music_disc_ocean.desc": "Ocean Waves",
+	"item.quark.music_disc_rain": "环境唱片",
+	"item.quark.music_disc_rain.desc": "Rainy Mood",
+	"item.quark.music_disc_wind": "环境唱片",
+	"item.quark.music_disc_wind.desc": "Heavy Wind",
+	"item.quark.music_disc_fire": "环境唱片",
+	"item.quark.music_disc_fire.desc": "Soothing Cinders",
+	"item.quark.music_disc_clock": "环境唱片",
+	"item.quark.music_disc_clock.desc": "Tick-Tock",
+	"item.quark.music_disc_crickets": "环境唱片",
+	"item.quark.music_disc_crickets.desc": "Cricket Song",
+	"item.quark.music_disc_chatter": "环境唱片",
+	"item.quark.music_disc_chatter.desc": "Packed Venue",
+	"item.quark.torch_arrow": "火把箭",
+	"item.quark.ancient_boat": "远古木船",
+	"item.quark.ancient_chest_boat": "远古木运输船",
+	"item.quark.ancient_fruit": "远古水果",
```

```suggestion
	"item.quark.ancient_fruit": "附魔水果",
```
此物品目前的英文名已经变为"Enchanted Fruit"。

---

请分清 1.19 和 1.20

---

抱歉，查看时没发现1.20有被折叠的部分，无视即可。